plugins/com.google.cloud.tools.eclipse.util/templates/appengine/pom.xml.standard.ftl

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Very similar to the flex `pom.xml` template. I'll highlight the difference below:

```diff
--- ./plugins/com.google.cloud.tools.eclipse.util/templates/appengine/pom.xml.flex.ftl  2017-06-05 17:34:41.314955620 -0400
+++ ./plugins/com.google.cloud.tools.eclipse.util/templates/appengine/pom.xml.standard.ftl      2017-06-12 11:44:09.308008253 -0400
@@ -14,8 +14,8 @@
     <appengine.maven.plugin.version>1.3.1</appengine.maven.plugin.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.7</maven.compiler.source>
+    <maven.compiler.target>1.7</maven.compiler.target>
     <maven.compiler.showDeprecation>true</maven.compiler.showDeprecation>
   </properties>
 
@@ -27,8 +27,8 @@
     <!-- Compile/runtime dependencies -->
     <dependency>
       <groupId>javax.servlet</groupId>
-      <artifactId>javax.servlet-api</artifactId>
-      <version>3.1.0</version>
+      <artifactId>servlet-api</artifactId>
+      <version>2.5</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
```